samples/apps/nobuiltins/nobuiltins.cpp

```diff
@@ -56,10 +56,8 @@ namespace nobuiltins
           }
 
           summary.quote_format = quote_info.format;
-          summary.quote =
-            fmt::format("{:02x}", fmt::join(quote_info.quote, ""));
-          summary.endorsements =
-            fmt::format("{:02x}", fmt::join(quote_info.endorsements, ""));
+          summary.quote = ds::to_hex(quote_info.quote);
```

Comment:
I can't seem to recall the reason why we encode these as hex and not base64, but it feels like the latter would be more consistent with the rest of binary data in CCF. @achamayou?

Reply:
I don't think there's a good reason other than this was done before we decided on the "hex for digests, base64 for everything else"-policy, it's definitely a good idea to change it.